plotly/tests/test_optional/test_figure_factory.py

```diff
@@ -672,3 +678,42 @@ def test_valid_endpts(self):
                                 tls.FigureFactory.create_scatterplotmatrix,
                                 df, use_theme=True, index='a',
                                 palette='Blues', endpts=[2, 1])
+
+    def test_scatter_plot_matrix_kwargs(self):
+
+        # check if test scatter plot matrix matches with
+        # the expected output
+
+        df = pd.DataFrame([[2, 'Apple'], [6, 'Pear'],
+                          [-15, 'Apple'], [5, 'Pear'],
+                          [-2, 'Apple'], [0, 'Apple']],
+                          columns=['Numbers', 'Fruit'])
+
+        test_scatter_plot_matrix = tls.FigureFactory.create_scatterplotmatrix(df, index='Fruit', endpts=[-10, -1],
+                                                                              diag='histogram', height=1000, width=1000,
+                                                                              size=13, title='Scatterplot Matrix',
+                                                                              use_theme=True, palette='YlOrRd',
+                                                                              marker=dict(symbol=136))
+
+        exp_scatter_plot_matrix = {'data': [{'marker': {'color': 'rgb(128.0, 0.0, 38.0)'},
+                                   'showlegend': False,
+                                   'type': 'histogram',
+                                   'x': [6, 5],
+                                   'xaxis': 'x1',
+                                   'yaxis': 'y1'},
+                                  {'marker': {'color': 'rgb(255.0, 255.0, 204.0)'},
+                                   'showlegend': False,
+                                   'type': 'histogram',
+                                   'x': [2, -15, -2, 0],
+                                   'xaxis': 'x1',
+                                   'yaxis': 'y1'}],
+                                   'layout': {'barmode': 'stack',
+                                              'height': 1000,
+                                              'showlegend': True,
+                                              'title': 'Scatterplot Matrix',
+                                              'width': 1000,
+                                              'xaxis1': {'anchor': 'y1', 'domain': [0.0, 1.0], 'title': 'Numbers'},
+                                              'yaxis1': {'anchor': 'x1', 'domain': [0.0, 1.0], 'title': 'Numbers'}}}
```

Comment:
pep8 again here, you should allow the original `'data'` to wrap onto a new line.